.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2017": true,
+        "greasemonkey": true
+    },
+    "extends": ["eslint:recommended", "plugin:compat/recommended"],
+    "parserOptions": {
+        "ecmaVersion": 9
+    },
+    "rules": {
+        "no-unused-vars": "off",
+        "no-undef": "off",
+        "no-extra-semi": "warn",
+        "no-useless-escape": "warn"
+    },
+    "settings": {
+        "polyfills": [
+            "String.prototype.includes",
+            "BroadcastChannel",
```

Comment:
Marking BroadcastChannel as polyfilled is probably the right thing to do, even though we aren't using (and don't need to use) a real polyfill, just a dummy.

Reply:
Oh, I was just quickly filling in whatever was in `polyfills.js`. Main purpose of specifying polyfills here is for stuff like `Array.prototype.flat` which are above the set ecmascript version and would throw an error if not specified.